cpp/src/join/hash_join.cu

```diff
@@ -47,34 +47,41 @@ namespace {
  * @brief Calculates the exact size of the join output produced when
  * joining two tables together.
  *
- * @throw cudf::logic_error if JoinKind is not INNER_JOIN or LEFT_JOIN
- *
- * @tparam JoinKind The type of join to be performed
+ * @throw cudf::logic_error if join is not INNER_JOIN or LEFT_JOIN
  *
  * @param build_table The right hand table
  * @param probe_table The left hand table
+ * @param preprocessed_build shared_ptr to cudf::experimental::row::equality::preprocessed_table for
+ * build_table
+ * @param preprocessed_probe shared_ptr to cudf::experimental::row::equality::preprocessed_table for
+ * probe_table
```

Comment:
Since we have enough time for the next release, I recommend to rework doxygen for this PR. The new line of each `@param` tag should be indented so it is cleaner to read.

```suggestion
 * @param preprocessed_build shared_ptr to cudf::experimental::row::equality::preprocessed_table for
 *        build_table
 * @param preprocessed_probe shared_ptr to cudf::experimental::row::equality::preprocessed_table for
 *        probe_table
```

Reply:
Done